CLAUDE.md

```
@@ -0,0 +1,62 @@
+poetry run pytest --cov=alkemio_virtual_contributor_engine --cov-fail-under=90 -v
+
+# Lint
+poetry run flake8 alkemio_virtual_contributor_engine/
+
+# Type check
+poetry run pyright alkemio_virtual_contributor_engine/
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)